src/main/java/uk/gov/hmcts/divorce/divorcecase/model/AlternativeService.java

```diff
@@ -108,4 +109,25 @@
     @CCD(access = {CaseworkerAccessBetaOnlyAccess.class})
     private Bailiff bailiff = new Bailiff();
 
+    @SuppressWarnings("PMD")
+    @JsonIgnore
+    public AlternativeServiceOutcome getOutcome() {
+        return AlternativeServiceOutcome.builder()
+            .alternativeServiceType(this.getAlternativeServiceType())
```

Comment:
is there any need to use getters here? maybe just use the local property

Reply:
Yes, I can change to the local properties (didn't think about it after I moved this method from the CaseData class)